src/mock/task.js

```diff
@@ -0,0 +1,72 @@
+import {getRandomInteger, COLORS, getBoolean} from "../utils.js";
+
+const generateDescription = () => {
+  const descriptions = [
+    `Изучить теорию`,
+    `Сделать домашку`,
+    `Пройти интенсив на соточку`
+  ];
+
+  const randomIndex = getRandomInteger(0, descriptions.length - 1);
+
+  return descriptions[randomIndex];
+};
+
+const generateDate = () => {
+  const isDate = Boolean(getRandomInteger(0, 1));
```

Comment:
Здесь надо использовать функцию
```suggestion
  const isDate = getBoolean();
```